plugins/extractors/maxcompute/maxcompute.go

```diff
@@ -0,0 +1,251 @@
+package maxcompute
+
+import (
+	"context"
+	_ "embed" // used to print the embedded assets
+	"sync"
+	"time"
+
+	client2 "github.com/alibabacloud-go/darabonba-openapi/v2/client"
+	maxcomputeclient "github.com/alibabacloud-go/maxcompute-20220104/client"
+	"github.com/goto/meteor/models"
+	v1beta2 "github.com/goto/meteor/models/gotocompany/assets/v1beta2"
+	"github.com/goto/meteor/plugins"
+	"github.com/goto/meteor/registry"
+	"github.com/goto/meteor/utils"
+	"github.com/goto/salt/log"
+	"google.golang.org/protobuf/types/known/anypb"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+type Config struct {
+	ProjectName     string `mapstructure:"project_name"`
+	EndpointProject string `mapstructure:"endpoint_project"`
+	AccessKey       struct {
+		ID     string `mapstructure:"id"`
+		Secret string `mapstructure:"secret"`
+	} `mapstructure:"access_key"`
+	SchemaName     string `mapstructure:"schema_name"`
```

Comment:
this should be optional, if not specified, please ensure the extractor fetch all tables from all schemas

Reply:
correct, please address this as our discussion before @Mayurjag 
you can get all schema from this
```
aliAccount := account.NewAliyunAccount(conf.AccessId, conf.AccessKey)
odpsIns := odps.NewOdps(aliAccount, conf.Endpoint)
project := odpsIns.Project(conf.ProjectName)
schemas := project.Schemas()
// show all schemas in the project
schemas.List(func(schema *odps.Schema, err error) {
	fmt.Println(schema.Name() + "\n")
})	
```

Reply:
Oh I just remember I always use odps in my local.. should we better to move to odps sdk now since that sdk that always developed by Ali team?

Reply:
because if I check the SDK that we're use rn: https://github.com/alibabacloud-go/maxcompute-20220104/ does NOT provide get all schemas API

Reply:
addressed